fuzzing/private/runtime/BUILD

```diff
@@ -0,0 +1,49 @@
+load(":clang_runtime_lib.bzl", "clang_runtime_lib")
+
+alias(
+    name = "asan",
+    actual = select({
+        "@platforms//os:linux": ":asan_linux",
+        "@platforms//os:macos": ":asan_macos",
+    }),
+    visibility = ["//visibility:public"],
+)
+
+alias(
+    name = "ubsan",
+    actual = select({
+        "@platforms//os:linux": ":ubsan_linux",
+        "@platforms//os:macos": ":ubsan_macos",
+    }),
+    visibility = ["//visibility:public"],
+)
+
+clang_runtime_lib(
+    name = "asan_linux",
+    basenames = [
+        # LLVM 15+
+        "libclang_rt.asan.so",
+        # LLVM 14 and earlier
+        "libclang_rt.asan-x86_64.so",
+    ],
+)
+
+clang_runtime_lib(
+    name = "asan_macos",
+    basenames = ["libclang_rt.asan_osx_dynamic.dylib"],
+)
+
+clang_runtime_lib(
+    name = "ubsan_linux",
+    basenames = [
+        # LLVM 15+
+        "libclang_rt.ubsan_standalone.so",
+        # LLVM 14 and earlier
+        "libclang_rt.ubsan_standalone-x86_64.so",
+    ],
+)
+
+clang_runtime_lib(
+    name = "ubsan_macos",
+    basenames = ["libclang_rt.ubsan_osx_dynamic.dylib"],
+)
```

Comment:
Should these rules always be defined? I believe Bazel supports specifying compatibility criteria: https://bazel.build/extending/platforms#skipping-incompatible-targets

Reply:
It does, but that doesn't factor into target selection, it just improves error messages and makes wildcard builds pass. Here, I had `tags = ["manual"]` in the macro, which was arguably too magic - I moved it out into the BUILD file.